docs/guides/aws-java-sdk-v2.md

```diff
@@ -22,9 +22,10 @@ The *S3 transfer manager* is a subsystem of SDK v2 that handles S3 uploads and d
 
 You can configure the concurrency and throughput of the S3 transfer manager manually using the `aws.client.maxConcurrency` and `aws.client.maxNativeMemory` configuration options. Alternatively, you can use the `aws.client.targetThroughputInGbps` option to set both values automatically based on a target throughput.
 
-## Multi-part uploads
+## Multi-part transfers
 
-Multi-part uploads are handled by the S3 transfer manager. You can use the `aws.client.minimumPartSize` and `aws.client.multipartThreshold` config options to control when and how multi-part uploads are performed.
+Multi-part transfer are handled by the S3 transfer manager. You can use the `aws.client.minimumPartSize` and `aws.client.multipartThreshold` config options to control when and how multi-part transfers are performed. 
+Concurrent multi-part downloads can consume large heap memory space due to the buffer size created per transfer. To avoid out of memory errors the size consumed by these buffers is limited to 400 MB. You can use `aws.client.maxDownloadBuffer` to change this value.
```

Comment:
```suggestion
Concurrent multi-part downloads can consume large heap memory space due to the buffer size created per transfer. To avoid out of memory errors, the size consumed by these buffers is limited to 400 MB. You can use `aws.client.maxDownloadBuffer` to change this value.
```